lib/board.rb

```diff
@@ -66,10 +66,53 @@ def tie?
   end
 
   # return formatted board for printing to terminal
-  def display_board 
-    "#{@header}\n#{row_1}\n#{row_2}\n#{row_3}\n#{row_4}\n#{row_5}\n#{row_6}"
+  def pretty_print
+    array = ["ABCDEFG\n"]
+    (1..6).to_a.each do |number|
+      ("A".."G").each do |letter|
+         array << @layout["#{letter}#{number}".to_sym].slot 
+      end
+      array << "\n"
+    end
+    array.join("")
+  end
+
+  def search_for_winner
+    return true if horizontal_win
+    return true if vertical_win
+    # return true if diag_win
```

Comment:
as we talked about i will fold the diagonal method into what we already have and test shortly

Reply:
Excellent, thank you! Great work today.